spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/Topology.scala

```diff
@@ -114,38 +108,27 @@ trait ModelSerializer extends GraphSerializable with TKerasSerializerHelper{
   }
 }
 
-class Sequential[T: ClassTag](val stopInferShape: Boolean = false)
-(implicit ev: TensorNumeric[T]) extends TSequential[T] {
-
-  override private[bigdl] def isCompatibleWithKeras(): Boolean = true
-
-  override private[bigdl] def isCompatibleWithTorch(): Boolean = false
+class Sequential[T: ClassTag]()
+(implicit ev: TensorNumeric[T]) extends KerasLayer[Activity, Activity, T] {
 
   private[bigdl] var frozen: Boolean = false
```

Comment:
Is frozen same as used?

Reply:
You can check if other layers are already used when they are added to the Sequential, yes?

Reply:
Let's think of bellowing scenario:
```
model = Sequential()
seq = Sequential()
model.add(seq)
seq.add(Dense(10)) // we should throw exception here, but "isUsed" is false for Dense(10) 
```